lib/mix/lib/mix/task.ex

```diff
@@ -275,6 +277,12 @@ defmodule Mix.Task do
     end
   end
 
+  defp output_task_debug_info(task, args) do
+    Mix.shell.info("Running 'mix #{task_to_string(task, args)}'")
```

Comment:
Can we do: `"** Running mix #{task_to_string(task, args)} (inside #{inspect proj})"`. I don t think we need to wrap in quotes the mix command, it would be just noise in this case. :)